cmake/toolchain/raspberrypi.cmake

```diff
@@ -11,13 +11,9 @@
 #
 # e.g. should the user install the tools in ``/home/user1` then the environment variable might be set using
 # `export RPI_TOOLCHAIN_DIR=/home/user/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/`
-#
-# **Note:** this toolchain file sets the system name to `arm-linux-gnueabihf` to be similar to other raspberry pi
-# toolchain files while providing a cleaner name of `raspberrypi`.  Its parallel platform file is therefore named
-# `arm-linux-gnueabihf.cmake` in the platforms directory.
 ####
 # Set system name
-set(CMAKE_SYSTEM_NAME "arm-linux-gnueabihf")
```

Comment:
We may also want to add `set(CMAKE_SYSTEM_PROCESSOR "arm")` so that if we want to add arm specific logic in the future we can.

Reply:
I think this is an adjustment we can make when we need this functionality.

Reply:
Changed my mind on this.  I'd like to set this.  It seems to make sense to me.